activiti-cloud-modeling-service/activiti-cloud-acceptance-tests-modeling/src/main/java/org/activiti/cloud/acc/modeling/steps/ModelingModelsSteps.java

```diff
@@ -207,7 +207,9 @@ private FormData getFormData(Model model) throws IOException {
 
     @Step
     public void checkCurrentModelValidation() throws IOException {
-        assertThat(validateCurrentModel()).extracting(Response::status).containsOnly(HttpStatus.SC_NO_CONTENT);
+        assertThat(validateCurrentModel())
+            .extracting(Response::status)
+            .containsOnly(HttpStatus.SC_NO_CONTENT, SC_BAD_REQUEST);
```

Comment:
This is a step used in acceptance tests to check a valid model (it is used on the step with `the model is valid` comment). We should not extend the expected response codes, since it can lead to false positive acceptance tests.